libsodium-sys/build.rs

```diff
@@ -427,4 +427,6 @@ fn build_libsodium() {
 
     println!("cargo:rustc-link-lib=static=sodium");
     println!("cargo:rustc-link-search=native={}/lib", install_dir);
+    println!("cargo:include={}/include", install_dir);
```

Comment:
Please add comments like: "To export env:DEP_SODIUM_INCLUDE", because one may delete these lines in the future, and our CI won't catch it.

Reply:
If I add a CI test that shouldn't be a problem. I can still add a comment though.

Reply:
No need if you will add a test